decidim-admin/app/forms/decidim/admin/user_group_csv_verification_form.rb

```diff
@@ -6,10 +6,20 @@ module Admin
     #
     class UserGroupCsvVerificationForm < Form
       include Decidim::HasUploadValidations
+      include Decidim::ProcessesFileLocally
```

Comment:
This concern is missing from 0.26 which is breaking the specs.

That was added at #9362 which relies on the dynamic file attachments (#9360).